src/main/java/com/salesforce/datacloud/jdbc/core/fsm/README.md

```diff
@@ -0,0 +1,23 @@
+```mermaid
+---
+config:
+  look: handDrawn
+---
+stateDiagram
```

Comment:
This is out of date and doesn't match State.java, but I'd like it to once this PR is out of "Draft"